src/test/scala/com/hometask/Task1Test.scala

```diff
@@ -0,0 +1,58 @@
+package com.hometask
+
+import org.scalatest.{Matchers, WordSpec}
+import com.hometask.Task1._
+
+import scala.concurrent.duration._
+import org.scalatest.concurrent.Timeouts
+import org.scalatest.time.{Seconds, Span}
+
+class Task1Test
+  extends WordSpec
+    with Matchers
+    with Timeouts {
```

Comment:
nit: It looks like `Timeouts` is deprecated.

Reply:
@deprecated("Please use org.scalatest.concurrent.TimeLimits instead")
trait Timeouts

right, missed that (